projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -222,9 +223,9 @@ export class DisplayEditComponent implements OnInit {
         const dialogRef =
             this._dialog.open<ConfirmationDialogComponent, ConfirmationDialogData>(ConfirmationDialogComponent, { data: dialogData});
 
-        dialogRef.afterClosed().subscribe((confirmed: boolean) => {
-            if (confirmed) {
-                this.deleteValue();
+        dialogRef.afterClosed().subscribe((payload: ConfirmationDialogPayload) => {
+            if (payload && payload.confirmed) {
+                this.deleteValue(payload.deletionComment);
```

Comment:
nothing, because the if condition checks if the payload exists. Originally I had it only checking if `payload.confirmed` was true, which did break when the payload was null. This fixed it.

Reply:
made optional in 050a4cd